src/lib.rs

```diff
@@ -22,20 +22,20 @@ use std::marker::PhantomData;
 use siphasher::reexports::serde;
 
 pub mod reexports {
+    pub use bitvec;
     #[cfg(feature = "random")]
-    pub use ::getrandom;
-    pub use bit_vec;
+    pub use getrandom;
+    pub use siphasher;
     #[cfg(feature = "serde")]
     pub use siphasher::reexports::serde;
-    pub use siphasher;
 }
 
 /// Bloom filter structure
 #[cfg_attr(feature = "serde", derive(serde::Serialize, serde::Deserialize))]
 #[cfg_attr(feature = "serde", serde(crate = "serde"))]
 #[derive(Clone, Debug)]
 pub struct Bloom<T: ?Sized> {
-    bit_vec: BitVec,
+    bit_vec: BitVec<u8>,
```

Comment:
By default it's `usize` but to keep compatibility with your api, had to use `u8`